tests/patterns/python/static_method.py

```diff
@@ -0,0 +1,15 @@
+class A:
+    #MATCH:
+    @staticmethod
+    def f():
+        pass
+
+    #MATCH:
+    @staticmethod()
+    def f():
+        pass
+
+    #OK:
+    @staticmethod("A ERROR")
```

Comment:
Perhaps make clear in the comment that this is _not_ a static method.

Reply:
Maybe not even valid Python? 🤷‍♂️

Reply:
Not valid python, just documenting the edge cases, but done.